README.rst

```diff
@@ -204,12 +204,12 @@ But normally, you shouldn't need this.
 Building libmpv
 ---------------
 
-You can enable building libmpv by enabling the configure option::
+libmpv is built by default, but you can disable it in the configure option::
 
-    printf "%s\n" -Dlibmpv=true > mpv_options
+    printf "%s\n" -Dlibmpv=false > mpv_options
 
-Note that this will make the mpv-build scripts also enable PIC for all used
-libraries. For this reason, be sure to run ``./clean`` before rebuilding.
+Note that building libmpv will also enable PIC for all used libraries, so be
```

Comment:
I don't see why we shouldn't just enable PIC in all cases

Reply:
Performances reasons allegedly. I have no idea if this matters for ffmpeg.

Reply:
considering linux distros are widely using PIC/PIE for years I call bullshit on this.
we should enable it by default.

Reply:
Works for me.

Reply:
This now nukes the `test-libmpv` script completely.